netmaster/objApi/apiController.go

```diff
@@ -79,6 +80,16 @@ func NewAPIController(router *mux.Router) *APIController {
 	return ctrler
 }
 
+// Utility function to check if string exists in a slice
+func stringInSlice(a string, list []string) bool {
```

Comment:
minor:
As an search optimization we can maintain a `map[string]bool` than `[]string`. But totally optional.